src/popup.js

```diff
@@ -164,7 +164,11 @@ chrome.runtime.onMessage.addListener((request) => {
         hideLoading();
       });
   } else if (request.prsUpdated) {
-    updatePRs();
+    updatePRs()
+      .then(()=>{
+        hideLoading();
+      });
```

Comment:
I don't think this then is necessary b/c the hideLoading function is called inside the updatePRs function.